### DISCUSSION TITLE
ability to move Upcoming Song Queue list

### DISCUSSION THREAD

**Discussion by bpglobalnetworks** [AzuraCast/AzuraCast#5560] [Ideas]
There will be some possibility that a way to move the songs that are in the queue list can be added, this especially when it is required to move a wrongly placed song. Thank you.

**Comment by BusterNeece** [comment]
@bpglobalnetworks Hello, currently this isn't possible, but you can delete any tracks that are supposed to play before what you wanted to queue so that it will play sooner. You can definitely submit this as a feature request over at https://features.azuracast.com/
